app/pages/lab/workflow.cjsx

```diff
@@ -261,6 +261,22 @@ EditWorkflowPage = React.createClass
 
           <hr />
 
+          {if 'worldwide telescope' in @props.project.experimental_tools
+            <div>
+              <div>
+                <AutoSave resource={@props.workflow}>
+                  <span className="form-label">Use World Wide Telescope API</span><br />
+                  <small className="form-help">Allow user to view subject in the WWT after classifying.</small>
+                  <br />
+                  <input type="checkbox" id="world_wide_telescope" onChange={@handleSetWorldWideTelescope} defaultChecked={@props.workflow.configuration?.external_api}/>
+                  <label htmlFor="world_wide_telescope">WorldWide Telescope</label>
```

Comment:
Convention is to wrap the input in its label. Defining an `id` doesn't make sense when building reusable components.